tests/dummy/app/templates/application.hbs

```diff
@@ -66,3 +66,19 @@
     {{/if}}
   {{/current-user}}
 </div>
+
+<div class='example' id='class-example'>
+  <h2>Class Example</h2>
+  <button {{action 'toggleModal2'}}>Toggle Another Modal</button>
+
+    {{code-snippet name='modal2.hbs'}}
```

Comment:
If you guys wouldn't mind putting your "examples" folder inside `tests/dummy/app/templates`, I'd be happy to add a PR to add a testing component like [this](https://github.com/truenorth/ember-cli-materialize/blob/5380de876bddba5899890dea70ffe3b3a75f53d8/tests/dummy/app/components/example-snippet.js) to both render the snippet as a partial, and render the `{{code-snippet}}` below. This would help ensure consistency between the visual example and the example code.